jcl/src/java.base/share/classes/java/lang/invoke/NativeMethodHandle.java

```diff
@@ -112,3 +113,25 @@ void compareWithNative(NativeMethodHandle left, Comparator c) {
 		c.compareStructuralParameter(left.vmSlot, this.vmSlot);
 	}
 }
+/*[ELSE] Panama */
+import java.util.List;
+import jdk.internal.invoke.NativeEntryPoint;
+
+class NativeMethodHandle extends MethodHandle {
+	public static MethodHandle make(NativeEntryPoint entryPoint, MethodHandle mh) {
+		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+		return null;
```

Comment:
This won't compile: the `return` here (and in `addRelatedMHs()`) is unreachable.